frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -800,10 +801,22 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 object: { name: featureFlag.key, id: featureFlag.id },
                 callback: () => {
                     featureFlag.id && actions.deleteFlag(featureFlag.id)
+                    // Load latest change so a backwards navigation shows the flag as deleted
+                    actions.loadFeatureFlag()
```

Comment:
i like the idea of adding some other visual indicator that a flag was deleted, tbh – the greyed out edit button is nice, but it'd be cool if there were some other indication.

Reply:
Ok yep I agree. Trying to get a little inspiration from Archived experiments: 

<img width="535" alt="Screenshot 2024-11-07 at 5 50 44 PM" src="https://github.com/user-attachments/assets/9db1aa59-58e0-4fe7-8027-de25fa93aedc">

Maybe something along the lines of this? 
<img width="719" alt="Screenshot 2024-11-07 at 5 52 05 PM" src="https://github.com/user-attachments/assets/764af3dc-8458-4c76-a594-041e45cda620">

Reply:
Oh actually, in the spirit of disabling edits for deleted flags, this probably makes the most sense? 
<img width="683" alt="image" src="https://github.com/user-attachments/assets/488943a4-ee7c-4433-aa34-ad0b90785499">

Reply:
Also for consideration: 
<img width="732" alt="image" src="https://github.com/user-attachments/assets/bfa45a1a-bdc4-413b-b150-a77d70b8a4a8">

The switch is disabled, but I think this is more confusing than the LemonBadge option

Reply:
Added the second option for now!

Reply:
second option is my favorite